_includes/footer.html

```diff
@@ -22,6 +22,7 @@
     </div>
 
     <!--End mc_embed_signup-->
+    <script async="" type="text/javascript" src="https://cdn.remotehq.net/remotehq.js?placement=lg.io" id="_remotehq_js" data-remotehqcompany="Envoy"></script>
```

Comment:
Additionally this could be `"Envoy,Productboard"` and it would show either Envoy or Productboard (it changes per request)